### DISCUSSION TITLE
Participate in the 2023 MNF mobile ecosystem survey!

### DISCUSSION THREAD

**Discussion by keith** [MobileNativeFoundation/discussions#209] [General]
tl;dr: please take the MNF's quick mobile ecosystem survey https://www.surveymonkey.com/r/MNF2023, results and raw data will be shared once they're available.

Hey everyone, recently the Mobile Native Foundation launched a [public survey](https://www.surveymonkey.com/r/MNF2023) to try and get a pulse on different trends throughout the mobile community. Specifically we're very interested in seeing how trends change based on things like company size, number of apps, and other things like that. We're hoping that these trends shed some light on areas where the community is moving, and where there could be more investment in open source projects. We plan to analyze the data from the survey and share the results here, as well as sharing the raw data for folks who want to do extra analysis.

**We can't do this without you!** So please spend a few minutes to fill out the survey https://www.surveymonkey.com/r/MNF2023

If you're interested in last year's results you can see those here: https://github.com/MobileNativeFoundation/survey-2022

Thanks!

**Comment by volkanbicer** [comment]
Hi @keith 

When can we expect to see the results?

**Reply by keith** [reply]
Thanks for the ping! We're actively trying to get them + the summary, we should have an update soon!
